scripts/zones/Yuhtunga_Jungle/mobs/Carthi.lua

```diff
@@ -15,18 +15,18 @@ end;
 -----------------------------------
 
 function onMobDeath(mob, player, isKiller)
+    if (isKiller) then -- Causes this to only run once per Carthi kill
```

Comment:
Old saber has an altogether other problem..Player gonna be nil. You aren't supposed to even engage it. So when it dies its not going to have a valid "owner" in the core.

Also see edit on my previous comment.